### PR TITLE
Vet clinic database: create animals table

### PR DESCRIPTION
**Pull request by TadesseDev**
## Changes introduced 😎
---
- Create tables in SQ ✔
- Insert data into a table ✔
- and query data in SQL. ✔

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /* Populate database with sample data. */
-
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals
+VALUES (DEFAULT, 'Agumon', '2020-2-3', 0, TRUE, 10.23),
```
Comment (mikethreels) [inline data.sql]: - [ ] inserting into a table should look something like below:
```
INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
   VALUES ('Agumon', '2020/2/3', '0', 'true', '10.23');
```

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Tadesse, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (TadesseDev) [reply data.sql]: @mikethreels Thanks for the review. I didn't know it was mandatory to do it that way. I'll make sure to do it the same way you do.

**Review by mikethreels — APPROVED** [review]
Hi Tadesse, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by TadesseDev** [comment]
@mikethreels Thank you.

**Pull request merged**